### PR TITLE
[v10] Backport of dependabot CVE updates

### PR DESCRIPTION
**Pull request by jentfoo**
Cherry-picked security updates that have been applied in master but failed to backport to v10

### CODE REVIEW

File: web/.cloudbuild/scripts/go.mod [hunk 1]
```diff
@@ -2,12 +2,14 @@ module github.com/gravitational/webapps/.cloudbuild/scripts
 
 go 1.17
 
+require github.com/gravitational/trace v1.1.15
```
Comment (jentfoo) [inline web/.cloudbuild/scripts/go.mod:5]: This style change occurred on master.  It's not important in this PR but we may want to return it to the require block below in master.

Comment (codingllama) [reply web/.cloudbuild/scripts/go.mod:5]: Could you run `go mod tidy` and commit the changes, if any? (If there are no changes then this is fine.)

**Review by codingllama — APPROVED** [review]

File: web/.cloudbuild/scripts/go.mod (hunk 1 above)
Comment (jentfoo) [reply web/.cloudbuild/scripts/go.mod:5]: No change from my system

**Comment by codingllama** [comment]
Please make sure every module is tidied with `go mod tidy`, other than that LGTM.

**Comment by jentfoo** [comment]
An additional version revert was required, but this PR should be good to review now.  Once merged I will update #23580 and #23582 so that only these packages are updated.

**Review by codingllama — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Pull request merged**